### PR TITLE
Fix crash

### PR DESCRIPTION
**Pull request by Waqar144**

### CODE REVIEW

File: src/helpers/qownnotesmarkdownhighlighter.cpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ void QOwnNotesMarkdownHighlighter::highlightMarkdown(const QString& text) {
                             text.startsWith(QLatin1String("```")) ||
                             text.startsWith(QLatin1String("~~~"));
     const QString &next = currentBlock().next().text();
-    const bool isHeading = text.at(0) == QChar('#');
+    const bool isHeading = !text.isEmpty() && text.at(0) == QChar('#');
```
Comment (pbek) [inline src/helpers/qownnotesmarkdownhighlighter.cpp]: is it really `text` and not `next`?

Comment (pbek) [reply src/helpers/qownnotesmarkdownhighlighter.cpp]: ah, yes. `text` seems ok :grin:

**Comment by pbek** [comment]
yes, fixed

File: src/helpers/qownnotesmarkdownhighlighter.cpp (hunk 1 above)
Comment (Waqar144) [reply src/helpers/qownnotesmarkdownhighlighter.cpp]: ;)

**Pull request merged**